### DISCUSSION TITLE
Is Card style missing from MaterialDesignTheme.Defaults?

### DISCUSSION THREAD

**Discussion by jbooth88** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2876] [Q&A] [answered]
Two questions:
MaterialDesignTheme.Card.xaml is not included in MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Defaults.xaml nor is it included in MaterialDesignThemes.Wpf;component/Themes/MaterialDesign3.Defaults.xaml

Is this intended? If so, are there any other styles defined in MaterialDesignThemes.Wpf that follow this rule? 
Also, should I be using MaterialDesign3 or is it not ready yet?

**Comment by ElieTaillard** [comment]
MaterialDesign3 is still in dev so it's not recommended to use it yet.

All styles from the demo app come from `<ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Defaults.xaml" /> `

So you should be able to use the Card style.

**Reply by jbooth88** [reply]
This is why I was asking, 

When I tried using `pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Defaults.xaml` I couldn't find the Card styles, and after some digging into the current branch available on GitHub, I found that some resource files (like Card and Badged) were not included. I wasn't sure if this was intended, because the control styles that were missing from MaterialDesignTheme.Defaults.xaml appear to be for controls specific to MaterialDesignXamlToolkit 

The Cards user control in MainDemo.Wpf project references `pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Card.xaml` specifically instead of referencing the MaterialDesignTheme.Defaults.xaml file. 


Thanks for the info on MaterialDesign3. Looks good so far!

**Reply by nicolaihenriksen** [reply, marked as answer]
@jbooth88 I believe that `MaterialDesignTheme.Defaults.xml` (and the others) do not include the MDIX "custom controls". They only include the overridden styles for the controls available in the BCL (e.g. `Button`, `ComboBox`, etc.).

For custom controls (e.g. `Card`, `RatingBar`, etc.), it is a common practice to place their default style in the (somewhat "magic") `Themes\Generic.xaml` file. They are however, as @Xaalek mentions, still available to be used.

https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/f7d9a42df36f57efa9521bf13a7701d003567a79/MaterialDesignThemes.Wpf/Themes/Generic.xaml#L11-L31
